app/models/concerns/user_authentication_concern.rb

```diff
@@ -13,6 +13,18 @@ module UserAuthenticationConcern
     after_create :create_instance_user
 
     include UserOmniauthConcern
+
+    # Overrides `Devise::Models::Validatable`
+    # This disables the devise email validation for system user.
+    def email_required?
```

Comment:
What I usually do is I have a separate module called ReplacementMethods, then I include that as the last thing inside included.